luajit-2.0/src/lua.h

```diff
@@ -9,6 +9,21 @@
 #ifndef lua_h
 #define lua_h
 
+#ifdef WIN32
+# ifndef LUA_WIN
+#  define LUA_WIN
+# endif
+# ifndef _WIN32
+#  define _WIN32
+# endif
+
+# if !defined LUA_LIB & !defined LUA_CORE & !defined luajit_c  &  !defined _LJ_ARCH_H &  !defined _LJ_DEF_H & !defined _LJ_OBJ_H
+#  pragma comment( lib, "libluajit.lib") 
```

Comment:
@hughperkins this pragma just links with the library - but only in certain cases (if module is not a part of the 'core' itself). It can be moved to the options - if somebody will implement these conditions in CMake